packages/web-comment/src/app.js

```diff
@@ -0,0 +1,94 @@
+const express = require('express');
+const compression = require('compression');
+const lusca = require('lusca');
+const path = require('path');
+const cookieParser = require('cookie-parser');
+const nunjucks = require('nunjucks');
+
+const pinoExpress = require('express-pino-logger');
+const uuid = require('uuid');
+require('express-async-errors');
+
+const serverConfig = require('./server.config');
+const logger = require('./logging/logger');
+const routes = require('./routes');
+
+const app = express();
+
+app.use(
+	pinoExpress({
+		logger,
+		genReqId: () => uuid.v4()
+	})
+);
+
+const isDev = app.get('env') === 'development';
+
+const nunjucksConfig = {
+	autoescape: true,
+	noCache: true,
+	watch: isDev,
+	express: app
+};
+
+// resolve the govuk-frontend module folder
+// */govuk-frontend/govuk/all.js -> */govuk-frontend/
+const govukFrontEndRoot = path.join(require.resolve('govuk-frontend'), '..', '..');
+
+const viewPaths = [
+	govukFrontEndRoot,
+	// */@ministryofjustice/frontend/moj/all.js -> */@ministryofjustice/frontend
+	path.join(require.resolve('@ministryofjustice/frontend'), '..', '..'),
+	// */@pins/common/src/index.js -> */@pins/common/src/frontend
+	path.join(require.resolve('@pins/common'), '..', 'frontend'),
+	path.join(__dirname, './'),
+	path.join(__dirname, 'public'),
+	path.join(__dirname, 'src'),
+	path.join(__dirname, 'hello-world'),
+	path.join(__dirname, 'views')
+];
+
+nunjucks.configure(viewPaths, nunjucksConfig);
+
+if (serverConfig.server.useSecureSessionCookie) {
```

Comment:
think this needs defining

Reply:
Just resolving for now to get the initial merge done then we can pick these points back up